tests/phpstan/bootstrap.php

```diff
@@ -0,0 +1,100 @@
+<?php
+// phpcs:ignoreFile
+
```

Comment:
This list looks like it will quickly go out of date and add maintenance burden. Can a more dynamic solution be found for this ?

Reply:
Good point but I think the maintenance burden is low. Constants are not introduced very often and this is not a complete list of all constants, just ones which are unconditionally accessed by the files that are scanned by PHPStan.